metrics-otlp/README.md

```diff
@@ -0,0 +1,134 @@
+
+   ```bash
+   helm upgrade metrics-agent open-telemetry/opentelemetry-collector --version 0.47.0 --install --namespace $KYMA_NS \
+     -f https://raw.githubusercontent.com/kyma-project/examples/main/metrics-otlp/metrics-agent-values.yaml \
+     --set config.exporters.otlp.endpoint=metrics-gateway.$KYMA_NS:4317
+   ```
+
+1. Verify the deployment
+   Check that the related Pod has been created in the Namespace and is in the `Running` state:
+   ```bash
+   kubectl -n $KYMA_NS rollout status daemonset metrics-agent
+   ```
+
+## Usage
+
+After you have followed the instructions you should have a setup in place as outlined in the diagram on top of the tutorial, with a gateway and an agent. By default, the node specific metrics are getting ingested by the agent automatically and pushed via the gateway to the configured backend. Furthermore, metrics for the gateway and agent instances itself will be exported.
+
+To add metrics ingestion for your custom workload you have two options:
+
+### Prometheus pull-based on base of annotations
+
+That approach assumes that you instrumented your application using a library like the [prometheus client library](https://prometheus.io/docs/instrumenting/clientlibs/), having a port in your workload exposed serving a typical prometheus metrics endpoint.
+
+The agent is configured with a generic scrape configuration which will determine endpoints to scrape in the cluster via annotations. You can either put the annotations to a service resolving your metrics port or directly at the pod itself. The annotations are as followed:
+
+```yaml
+prometheus.io/scrape: "true"   # mandatory to enable automatic scraping
+prometheus.io/scheme: https    # optional, default is "http" if no Istio sidecar is used. When using a sidecar (Pod has label `security.istio.io/tlsMode=istio`), the default is "https". Use "https" to scrape workloads using Istio client certificates.
+prometheus.io/port: "1234"     # optional, configure the port under which the metrics are exposed
+prometheus.io/path: /myMetrics # optional, configure the path under which the metrics are exposed
+```
+
+Having the annotations in place is everything you need, metrics ingestion should start automatically.
+
+Be aware of, that the agent is capable of scraping endpoints even if the workload is using istio and accepts only mTLS communication. As the agent itself should not be part of the Service Mesh in order to observe the Service Mesh, the agent uses a sidecar but having no traffic interception enabled at all. Instead it mounts the client certificate and uses the certificate natively for the communication. That is a [recommended approach](https://istio.io/latest/docs/ops/integrations/prometheus/#tls-settings) by Istio itself.
```

Comment:
```suggestion
> **NOTE:** The agent can scrape endpoints even if the workload uses Istio and accepts only mTLS communication. Because the agent itself should not be part of the Service Mesh in order to observe the Service Mesh, the agent uses a sidecar but has no traffic interception enabled. Instead, it mounts the client certificate and uses the certificate natively for the communication. That is a [recommended approach](https://istio.io/latest/docs/ops/integrations/prometheus/#tls-settings) by Istio.
```